README.md

```diff
@@ -438,22 +446,78 @@ _To be added_
 
 ### 3. Other configuration options
 
-SGX-LKL-OE has a number of other configuration options e.g. for configuring the
-in-enclave scheduling, network configuration, or debugging/tracing. To see all
-options, run:
+SGX-LKL-OE has a number of other configuration options, e.g. for configuring the
+disks, in-enclave scheduling, network configuration, or debugging/tracing.
+
+There are two ways to specify configuration options: environment variables and JSON files.
+
+During debugging, it can be convenient to specify debugging options as environment
+variables. Note that for debugging options to have an effect, SGX-LKL must be built
+with `DEBUG=true`.
+To see a list of all environment configuration variables, run:
 ```
-sgx-lkl-run-oe --help
+sgx-lkl-run-oe --help-config
 ```
 
-Note that for the debugging options to have an effect, SGX-LKL must be built
-with `DEBUG=true`.
+The recommended way to configure SGX-LKL is through host config and app config JSON files.
+Using JSON files is also required when creating Confidential Containers (see next section).
+To create initial host and app config files in the current directory, run:
+```
+sgx-lkl-cfg create --disk sgxlkl-disk.img
+```
+Note that you may have to adapt the generated files to your needs.
+See the `tests/containers/cc` folder for example host and app config files.
+You can then run SGX-LKL using the host and app config files alone:
+```
+sgx-lkl-run-oe --hw-debug --host-config=host-config.json --app-config=app-config.json
+```
+Note that environment variables can still be used to specify additional options.
+This is especially useful for host configuration options that are only known
+at runtime in the target system where SGX-LKL will run, for example the name
+of the network tap device (`SGXLKL_TAP`).
+
+F. Confidential Container bundling for deployment
+-------------------------------------------------
+
+A Confidential Container is a Docker image that only contains the disk images
+and the host and app config files.
```

Comment:
Why would the CC contain the host_config? By definition, the host_config should be provided by the deployment environment, no?

Reply:
Some things are known at CC build time, like the paths of the disks.

Reply:
I wonder if it wouldn't be simpler if we always just called the disks, `disk0.img`, `disk1.img` etc. and the CC provided all of them in that order. Is there a benefit of giving the container builder control over this? One benefit is that the host_config gives the container builder a mechanism to configure the host environment, and we may need this in the future.

Reply:
For CCs this may work but I'd like to keep the flexibility of specifying explicit paths in SGX-LKL itself since it may be used in cases that are not strictly CCs, like in our Azure ML pipeline demo at //build.